sample-apps/delivery-customizations-sample-app/web/index.js

```diff
@@ -152,12 +152,7 @@ export async function createServer(
   }
 
   function normalizeCustomization({ id, metafield, ...customization }) {
-    return Object.assign(
-      {},
-      customization,
-      { id: gidToId(id) },
-      metafield
-    );
+    return Object.assign({}, customization, { id: gidToId(id) }, metafield);
```

Comment:
Nit: Could this be replaced with:

```suggestion
    return { ...customization, id: gidToId(id), metafield };
```